src/ImageCropper.tsx

```diff
@@ -21,6 +22,7 @@ interface IProps {
   areaColor?: string;
   areaOverlay?: ReactNode;
   setCropperParams: (params: ICropperParams) => void;
+  cropArea?: IConstraints;
```

Comment:
This is probably misleading compared to `cropAreaWidth` and `cropAreaHeight`. I would prefer rename `cropAreaHeight` to `height`, and `cropAreaWidth` to `width`.